system_metrics_collector/src/system_metrics_collector/linux_cpu_collector.cpp

```diff
@@ -0,0 +1,47 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+#include <memory>
+
+#include "rclcpp/rclcpp.hpp"
+#include "rcutils/logging_macros.h"
+
+#include "../../src/system_metrics_collector/constants.hpp"
+#include "../../src/system_metrics_collector/linux_cpu_measurement_node.hpp"
```

Comment:
Any reason for the relative includes? I see a `target_link_libraries` so you shouldn't need these. Does it need a `target_include_directories`?

Reply:
It looks like these headers shouldn't be exposed to the public.